modules/015-admission-policy-engine/crds/doc-ru-security-policy.yaml

```
@@ -66,116 +66,117 @@ spec:
                     allowedAppArmor:
                       type: array
                       description: |
-                        Список допустимых к использованию контейнерами профилей AppArmor.
+                        Список профилей AppArmor для контейнеров.
                       example: ["runtime/default", "unconfined"]
                       items:
                         type: string
-                        description: "Профиль AppArmor."
+                        description: Профиль AppArmor.
                     allowedFlexVolumes:
-                      description: "Список допустимых к использованию драйверов Flex Volumes."
+                      description: Список драйверов Flex Volumes.
                       items:
                         properties:
                           driver:
-                            description: "Название драйвера."
+                            description: Название драйвера.
                     allowedUnsafeSysctls:
                       description: |
-                        Список небезопасных ключей sysctl, доступных к использованию.
+                        Список доступных небезопасных ключей sysctl.
 
-                        Используйте `*` для разрешения всех ключей.
+                        Используйте `*`, чтобы разрешить всех ключей.
```

```suggestion
                        Используйте `*`, чтобы разрешить всех ключей.
```
@elenashliaga   ?????

---

```suggestion
                        Используйте `*`, чтобы разрешить все ключи.
```